Hymns/Browse/TagListView.swift

```diff
@@ -11,11 +11,18 @@ struct TagListView: View {
 
     var body: some View {
         VStack {
+            if viewModel.tags.isEmpty {
+                VStack {
+                    Image("empty tag illustration")
+                    Text("Create tags by tapping on the tag icon on any hymn").padding().multilineTextAlignment(.center)
```

Comment:
I talked to Emily about this, but I think it's better to include the tag icon in the message, since it's not super clear from the graphic. can you change this to `Create tags by tapping on the TAG_ICON in any hymn`